.env.template

```diff
@@ -1,6 +1,6 @@
 PORT=8080
 
-DATABASE_URL="postgres://postgres:permanent@localhost:5432/test_permanent"
+DATABASE_URL="postgres://postgres:permanent@database:5432/test_permanent"
```

Comment:
I think this needs to be `permanent` now, not `test_permanent`, based on our conversations. Thanks again for going through all this with me!